changeuser.php

```diff
@@ -0,0 +1,184 @@
+<?php
+include "include/header.php"; ?>
+
+<div style="text-align:center">
+<?php
+
+ob_implicit_flush(true);
+ob_end_flush();
+
+function flog($msg) {
+	echo $msg."<br>";
+	ob_flush();
+	flush();
+}
+
+function flogb($msg) {
+	echo "<b>".$msg."</b><br>";
+	ob_flush();
+	flush();
+}
+function flogerror($msg) {
+	echo "<font color='red'><b>".$msg."</b></font>"."<br>";
+	ob_flush();
+	flush();
+}
+
+
+//Create unique jobid
+$jobid = date('Ymd-H-i-s');
+$failed = false;
+
+##################### 1. FETCH VALIDATED FORM INPUT ##############################
+//flog("Reading fields...");
+
+$email = $_REQUEST["email"];
+$password = $_REQUEST["password"];
+$password_r = $_REQUEST["pasword_r"];
+$firstname = $_REQUEST["realname_1"];
+$lastname = $_REQUEST["realname_2"];
+$schoolname = $_REQUEST["schoolname"];
+$scholadress = $_REQUEST["schooladress"];
+$studentstotal = $_REQUEST["studentstotal"];
+$studentsyear = $_REQUEST["studentsyear"];
+$mode = $_REQUEST["mode"];
+$teamname = $_REQUEST["teamname"];
+$new_teamname = $_REQUEST["new_teamname"];
+$newname= $_REQUEST["newname"];
+$user = $_REQUEST["user"];
+$class = $_REQUEST["class"];
+$action = $_REQUEST["action"];
+
+$password_hashed = password_hash($password, PASSWORD_BCRYPT); //hash password
+
+##################### 2. MODE 'ADD': CREATE USER / CLASS #######################
+
+## 2a Create user
+$teacherid = -1;
+if ($mode == "ADD"){
```

Comment:
Warum ADD in einer changeuser.php? Mir wäre eine Aufteilung in drei Dateien lieber (eine für add und eine für change und eine für new)